roboteam_ai/include/roboteam_ai/control/positionControl/PositionControlUtils.h

```diff
@@ -5,42 +5,64 @@
 #ifndef RTT_POSITIONCONTROLUTILS_H
 #define RTT_POSITIONCONTROLUTILS_H
 
+#include "interface/api/Output.h"
 #include "roboteam_utils/Vector2.h"
-#include "utilities/Constants.h"
+#include "utilities/StpInfoEnums.h"
 
 namespace rtt::ai::control {
 
 class PositionControlUtils {
-   private:
-    static constexpr double MAX_TARGET_DEVIATION = 0.05;
+   public:
+    constexpr static const double TIME_STEP = 0.1;
+    constexpr static const int COLLISION_DETECTOR_STEP_COUNT = 10;
 
-    // minimum distance needed to consider the current target reached
-    static constexpr double MIN_DISTANCE_TARGET_REACHED = 2 * Constants::ROBOT_RADIUS();
+    constexpr static const double MIN_ENEMY_MOVING_ROBOT_DISTANCE = 3 * ai::Constants::ROBOT_RADIUS_MAX();
+    constexpr static const double MIN_ENEMY_STALE_ROBOT_DISTANCE = 2 * ai::Constants::ROBOT_RADIUS_MAX();
+
+    constexpr static const double MIN_OUR_ROBOT_DISTANCE_MOVING = 3 * ai::Constants::ROBOT_RADIUS_MAX();
+
+    constexpr static const double POSITION_TOLERANCE_DISTANCE = 0.05;
+    constexpr static const double MAX_STALE_VELOCITY = 0.05;
```

Comment:
Add doxygen comments. what is the constant for and why is this value given? 

Also, why is this constexpr? wouldn't just const be enough?